### DISCUSSION TITLE
Proposal: Two very important field parameters

### DISCUSSION THREAD

**Discussion by miagg** [oneduo/nova-file-manager#130] [Ideas]
I really miss two very important field parameters I had on my previous Nova 3 FileManager.

1. `folder` to navigate to a specific folder within the disk (not locking, just navigating as default folder)
2. `filter` or something similar to only show specific file types (e.g. images)

This will help us organize the content and also provide better experience for the content management users.
I know there is a [on-demand filesystem](https://oneduo.github.io/nova-file-manager/access-control.html#on-demand-filesystem) for my first request but it's mainly for isolation which is overkill for what I need.

I hope there are plans to add them soon.

Thanks again!

**Comment by Rezrazi** [comment]
These are interesting features, however it might be tricky for us to include them in our plans as we're just a two-person-team with full time jobs. Sometimes it is not possible for us to work on the project as much as we want to.

We would gladly accept a PR for this.

**Reply by miagg** [reply]
> These are interesting features, however it might be tricky for us to include them in our plans as we're just a two-person-team with full time jobs. Sometimes it is not possible for us to work on the project as much as we want to.
> 
> We would gladly accept a PR for this.

I truly understand. There is no pressure, just adding stuff here to consider for your future roadmap.

I would love to help later on for some of them.
